docs/journalist.rst

```diff
@@ -286,11 +286,10 @@ delete the files from your device:
 To decrypt and view documents or messages, return to your **Persistent** folder.
 All key actions are initiated by double-clicking:
 
-- Double-clicking archives in ZIP or gzip format will open the "File Roller"
-  application, which allows you to extract the contents.
+- Double-clicking archives in ZIP or gzip format will open the "Archive Manager"
+  application (called "file-roller" on the command line), which allows you to extract the contents.
```

Comment:
Use ``file-roller`` instead? command line application names tend to be formatted as code.

Reply:
fixed in 1c7e1817a